rmf_task_msgs/srv/CancelTask.srv

```diff
@@ -0,0 +1,12 @@
+# Submit Task | "POST" service call
```

Comment:
This comment looks like a copy/paste error.

Reply:
Fixed https://github.com/osrf/rmf_core/commit/d3880be4738600ed8a62f9fd023732fcb6b670cc